internal/btpcli/client.go

```diff
@@ -310,12 +310,13 @@ func (v2 *v2Client) Execute(ctx context.Context, cmdReq *CommandRequest, options
 	}
 
 	if cmdRes.StatusCode >= 400 {
+
 		var backendError struct {
-			Message string `json:"error"`
+			Message     string `json:"error"`
 		}
 
 		if err = json.NewDecoder(res.Body).Decode(&backendError); err == nil {
-			err = fmt.Errorf(backendError.Message)
+			err = fmt.Errorf("error: %s", backendError.Message)
```

Comment:
As you changed this error message logging you will probably also need to adjust some test cases with respect to the expected error message format

Reply:
I've reverted this change for the time being